### ISSUE TITLE
awk doc: array initialization: another option

### ISSUE THREAD

**Issue by dtkerns** [adambard/learnxinyminutes-docs#3164]
The document says:
```
    # Arrays
    arr[0] = "foo"
    arr[1] = "bar"
    # Unfortunately, there is no other way to initialize an array. Ya just
    # gotta chug through every value line by line like that.
```

Not true, here's another option:
```
split("foo,bar,baz", arr, ",");
```
I actually like to assign the return value from split() into a variable too.

Also at the very top it says: "It has a C-like syntax, but without semicolons,"
While semicolons are not **required**, it's a good practice to use them.

**Comment by dtkerns** [comment]
note: split() will use indexes 1-N, not 0-(N-1), consistent with the auto splitting of records into $1 - $NF

**Comment by divayprakash** [comment]
@dtkerns Could you try to make a PR for the same?

**Comment by dtkerns** [comment]
I'm not sure I know the mechanics of github to do a pull request... guess I need to go read a tutorial :|

**Comment by divayprakash** [comment]
@dtkerns I'd be happy to help you however possible. Let me know if you need anything. Give the [contributions guide](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown) a read.

**Comment by dtkerns** [comment]
hope I did it right!

**Comment by divayprakash** [comment]
PR #3215

**Comment by divayprakash** [comment]
Closed in #3215
